coding-concepts.js

```diff
@@ -47,22 +47,28 @@ const myCodingSkills = {
   databases: "PostgreSQL",
   versionControl: "GitHub"
 }
-// console.log(myCodingSkills.languages[0])
+console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:Javascript
+// b) Verify and explain: When we console.log(), we call on Object myCodingSkills. Then by using dot notation, we call on languages, then by using bracket notation we call on Javascript who's index is 0.
 
 // --------------------STRETCH: What will this log?
 
-class Learn {
+class Learn {                     
   constructor(name) {
     this.student = name
     this.cohort = "Delta"
     this.year = 2023
   }
 }
 const learnStudent = new Learn("George")
-// console.log(learnStudent)
-
-// a) Your answer:
-// b) Verify and explain:
+console.log(learnStudent)
+
+// a) Your answer: class Learn {
+//   constructor(name) {
+//     this.student = name
+//     this.cohort = "Delta"
+//     this.year = 2023
+//   }
+// }
+// b) Verify and explain: It logged -> Learn { student: 'George', cohort: 'Delta', year: 2023 }. I believe it passed the string "George" into the name, since it was a value with no designation.
```

Comment:
The `learnStudent` variable is holding a new instance of the class Learn. This means it is creating a new object, with one argument ("George"). We need to pass this argument because the constructor in the class is expecting 1 argument, which is for the name. The other two keys have default values, so they don't need to be specified.